src/dune_rules/lib.ml

```diff
@@ -372,6 +372,7 @@ end
 
 type db =
   { parent : db option
+  ; db_host : db Lazy.t
```

Comment:
Let's avoid being cute here, `db option` is enough.

Also, the field should be named `host` for consistency.

Reply:
Did that in the latest push